x/oracle/genesis.go

```diff
@@ -40,16 +40,30 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	for _, elem := range genState.StakerInfosAssets {
 		k.SetStakerInfos(ctx, elem.AssetId, elem.StakerInfos)
 	}
+	// set validatorReportInfos
+	for _, elem := range genState.ValidatorReportInfos {
+		k.SetValidatorReportInfo(ctx, elem.Address, elem)
+	}
+	// set vlidatorMissedRounds
+	for _, elem := range genState.ValidatorMissedRounds {
+		for _, missedRound := range elem.MissedRounds {
+			k.SetValidatorMissedRoundBitArray(ctx, elem.Address, uint64(missedRound.Index), missedRound.Missed)
```

Comment:
Why not directly define this **Index** as uint64 in the proto file for this type conversion?